src/js/angular/import/templates/import-resource-tree.html

```diff
@@ -128,7 +135,22 @@
             <td class="import-resource-cell import-resource-cell-select"></td>
             <td ng-style="{'padding-left': (filterByType !== TYPE_FILTER_OPTIONS.FILE ? resource.indent : '8px')}"
                 class="import-resource-cell import-resource-message">
-                {{ resource.importResource.message || ''}}
```

Comment:
split the whole thing in separate component maybe

Reply:
Done